src/app/App/MainScreen/StatusBar/buttons/useImportNotes.tsx

```diff
@@ -253,11 +317,20 @@ export const useImportNotes = ({
 
 			if (tagId === null && filenameBasePathSegments.length > 0) {
 				const tagNameToCreate = filenameBasePathSegments.join('/');
-				tagId = await tagsRegistry.add(tagNameToCreate, null);
-				tagsChanged();
+				if (tagNameToCreate.trim()) {
```

Comment:
Reuse trimmed value or check not trimmed value. Keep validations and used values consistent